File: insight/src/test/testData/java/SimplifyBranch.java
public class SimplifyBranch {
    public void simplifyBranch() {
        if (true) {
            Thread.sleep(100);
        }
        Thread.sleep(100);
    }

    public void simplifyBranch2() {
        Thread.sleep(100);
        if (true) {
            Thread.sleep(100);
        }
        Thread.sleep(100);
    }
}

File: insight/src/test/testData/kotlin/SimplifyBranch.kt
public class SimplifyBranch {
    fun simplifyBranch() {
        if (true) {
            Thread.sleep(100)
        }
        Thread.sleep(100)
    }

    fun simplifyBranch2() {
        Thread.sleep(100)
        if (true) {
            Thread.sleep(100)
        }
        Thread.sleep(100)
    }
}